slick/src/main/scala/slick/basic/BasicBackend.scala

```diff
@@ -228,7 +230,7 @@ trait BasicBackend { self =>
     protected[this] def runSynchronousDatabaseAction[R](a: SynchronousDatabaseAction[R, NoStream, This, _], ctx: Context, highPrio: Boolean): Future[R] = {
       val promise = Promise[R]()
       ctx.getEC(synchronousExecutionContext).prepare.execute(new AsyncExecutor.PrioritizedRunnable {
-        def highPriority = highPrio
+        def priority = ctx.priority(!highPrio)
```

Comment:
So `highPrio` now means `MediumPriority`.  Should this flag be called `mediumPrio` then?